src/model.rs

```diff
@@ -1298,6 +1310,22 @@ mod tests {
         assert_eq!(model.n_lp_iterations(), 0);
     }
 
+    #[test]
+    fn set_memory_limit() {
+        let model = Model::new()
+            .hide_output()
+            .set_memory_limit(0)
+            .include_default_plugins()
+            .read_prob("data/test/simple.lp")
+            .unwrap()
+            .solve();
+        let status = model.status();
+        assert_eq!(status, Status::MemoryLimit);
+        assert!(model.solving_time() < 0.5);
```

Comment:
This check should be okay but I'd rather not check in a hard-coded value for checking time. Seems slightly fragile, and I have a feeling I'm going to regret it later 😄 
```suggestion
```